### DISCUSSION TITLE
[Feature Request]: Open port 8085 of cluster-autoscaler to access "/metrics" via kube.tf

### DISCUSSION THREAD

**Discussion by zypriafl** [kube-hetzner/terraform-hcloud-kube-hetzner#797] [Ideas]
### Description

Hello,

We use the cluster-autoscaler feature. Its working well. 

Know we want to keep track of the node historic node count in Prometheus and Grafana.

We are wondering what is the best way to do it.
We see that the metrics are server on port 8085 via "/metrics".
However the Port is not opened by default and no Service is connected.

**Comment by mysticaltech** [comment]
@zypriafl You could definitely use the `extra_firewall_rules` variable to open (or close) any ports even after the cluster is up.

**Comment by mysticaltech** [comment]
@kube-hetzner/core Apart from the firewall port, that is not an issue, if you have any guidance for Prometheus / Grafana, I would be very curious too.

@zypriafl from what I can see, it seems super quick to deploy via helm, have a look at https://www.davidprat.com/install-k3s-prometheus-and-grafana-in-5-minutes/.
